p2p_wallet/Scenes/Main/Send/SendViaLink/SendCreateLinkCoordinator.swift

```diff
@@ -84,9 +87,13 @@ final class SendCreateLinkCoordinator: Coordinator<SendCreateLinkCoordinator.Res
         navigationController.present(av, animated: true)
     }
     
-    private func showErrorView() {
+    private func showErrorView(error: Error) {
         let view = SendCreateLinkErrorView { [unowned self] in
-            result.send(.error)
+            if (error as NSError).isNetworkConnectionError {
+                result.send(.networkError)
+            } else {
+                result.send(.normal)
```

Comment:
false behavior:
- For networkConnectionError just don't show error scene and return directly to Send.
- For other error, result can NOT be normal, in other words, for other network you should move to error view, when click try again go back to Send.
So in general, for ANY ERROR, user must return to Send, not Home
Please read the requirement again